docs/spectral_cube.rst

```diff
@@ -85,7 +85,7 @@ spectral regions from the cube.
 
     >>> import astropy.units as u
     >>> from specutils.manipulation import spectral_slab
-    >>> ss = spectral_slab(sc, 5000.*u.AA, 5003.*u.AA)
+    >>> ss = spectral_slab(sc, 5000.*u.AA, 5003.*u.AA) # doctest: +REMOTE_DATA
     >>> ss.shape  #doctest:+SKIP
```

Comment:
I don't know why even set up the test with remote data if you then skip the important stuff from doctest... But that is a discussion for another day.